Library/Services/Implementation/NfieldSurveyInterviewSimulationService.cs

```diff
@@ -34,6 +55,65 @@ public async Task<Uri> GetHintsAsync(string surveyId)
             }
         }
 
+        /// <summary>
+        /// <see cref="INfieldSurveyInterviewSimulationService.StartSimulationAsync(string, InterviewSimulation)"/>
+        /// </summary>
+        public Task<InterviewSimulationResult> StartSimulationAsync(string surveyId, InterviewSimulation simulationRequest)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(surveyId, nameof(surveyId));
+            Ensure.ArgumentNotNull(simulationRequest, nameof(simulationRequest));
+
+            var content = MultipartDataContent(simulationRequest);
+
+            return Client.PostAsync(StartInterviewSimulationsEndPoint(surveyId), content)
```

Comment:
we want to move away from this syntax. It is better to keep it as you had it in previous commits (using awaits)

Reply:
Do you mean not use ContinueWith?